base/sort.jl

```diff
@@ -682,7 +682,7 @@ end
 function radix_sort!(v::AbstractVector{U}, lo::Integer, hi::Integer, bits::Unsigned,
                      t::AbstractVector{U}, chunk_size=radix_chunk_size_heuristic(lo, hi, bits)) where U <: Unsigned
     # bits is unsigned for performance reasons.
-    mask = UInt(1) << chunk_size - 0x1
+    mask = UInt(1) << chunk_size - 1
```

Comment:
Bitshift has [higher precedence](https://docs.julialang.org/en/v1/manual/mathematical-operations/#Operator-Precedence-and-Associativity) than subtraction, so this should change neither the value nor type of the bitshift distance. Perhaps it would be more clear to write:
1
```suggestion
    mask = (UInt(1) << chunk_size) - 1
```
2
```suggestion
    mask = UInt(1)<<chunk_size - 1
```
or
3
```suggestion
    mask = (UInt(1)<<chunk_size) - 1
```